.github/pull.yml

```diff
@@ -0,0 +1,5 @@
+version: "1"
```

Comment:
very non-ideal, but we should fitler this out...
https://stackoverflow.com/questions/28703140/pull-request-ignore-some-file-changes

Reply:
removed from this branch, seems like it will have to be manually removed each time :/